build-system/tasks/default-task.js

```diff
@@ -86,4 +86,5 @@ defaultTask.flags = {
     "  Doesn't use nailgun to invoke closure compiler (much slower)",
   version_override: '  Overrides the version written to AMP_CONFIG',
   custom_version_mark: '  Set final digit (0-9) on auto-generated version',
+  host: 'Host to serve the project on. localhost by default.',
```

Comment:
This breaks indentation in `gulp help`. Add two spaces at the start of the string.

Reply:
Also, might as well add a flag for `argv.port`.

Reply:
#27674